diff --git a/mind-wave.el b/mind-wave.el
index cdb5a36..7eadf12 100644
--- a/mind-wave.el
+++ b/mind-wave.el
@@ -377,9 +377,6 @@ Then Mind-Wave will start by gdb, please send new issue with `*mind-wave*' buffe
   "Mind Wave Chat Keymap")
 
 (define-derived-mode mind-wave-chat-mode gfm-mode "Mind-Wave"
-  (setq-local markdown-hide-markup markdown-hide-markup-in-view-modes)
-  (setq-local markdown-fontify-code-blocks-natively t)
-  (add-to-invisibility-spec 'markdown-markup)
   (when mind-wave-auto-update-old-chats
     (mind-wave--update-chat-buffer-to-new-version)))
 
diff --git a/mind_wave.py b/mind_wave.py
index 8f0d8b4..f8ba432 100755
--- a/mind_wave.py
+++ b/mind_wave.py
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#! /usr/bin/env nix-shell
+#! nix-shell -i python3 -p python3Packages.openai python3Packages.epc python3Packages.sexpdata python3Packages.six
+#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/f3a0b6fd70a796c901a0e4f8209356f52bda96be.tar.gz
 # -*- coding: utf-8 -*-
 
 # Copyright (C) 2023 Andy Stewart
